HackIllinois/Misc/Log.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Log.swift
+//  HackIllinois
+//
+//  Created by Tommy Yu on 1/4/19.
+//  Copyright © 2019 HackIllinois. All rights reserved.
+//  This file is part of the Hackillinois iOS App.
+//  The Hackillinois iOS App is open source software, released under the University of
+//  Illinois/NCSA Open Source License. You should have received a copy of
+//  this license in a file with the distribution.
+//
+
+import Foundation
+import os.log
+
+struct Logger {
+    static var viewController = OSLog(subsystem: "com.hackillinois.ios", category: "view_controller")
```

Comment:
I like the view controller log, but I think it should be for UI/autolayout issues, whereas API calls should probably have their own log.